src/pages/settings/Payments/ChooseTransferAccountPage.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import ROUTES from '../../../ROUTES';
+import HeaderWithCloseButton from '../../../components/HeaderWithCloseButton';
+import ScreenWrapper from '../../../components/ScreenWrapper';
+import Navigation from '../../../libs/Navigation/Navigation';
+import styles from '../../../styles/styles';
+import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import KeyboardAvoidingView from '../../../components/KeyboardAvoidingView/index';
+import PaymentMethodList from './PaymentMethodList';
+import ONYXKEYS from '../../../ONYXKEYS';
+import compose from '../../../libs/compose';
+import * as paymentPropTypes from './paymentPropTypes';
+import * as PaymentMethods from '../../../libs/actions/PaymentMethods';
+import * as PaymentUtils from '../../../libs/PaymentUtils';
+
+const propTypes = {
+    walletTransfer: paymentPropTypes.walletTransferPropTypes,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    walletTransfer: {},
+};
+
+class ChooseTransferAccountPage extends React.Component {
+    constructor(props) {
+        super(props);
+        this.paymentMethodSelected = this.paymentMethodSelected.bind(this);
+    }
+
+    /**
+     * Go back to TransferPage with the selected bank account
+     *
+     * @param {Object} nativeEvent
+     * @param {String} account
+     */
+    paymentMethodSelected(nativeEvent, account) {
+        if (!account) {
+            PaymentUtils.addPaymentMethodType(this.props.walletTransfer.filterPaymentMethodType);
+            return;
+        }
+        PaymentMethods.updateWalletTransferData({selectedAccountID: account});
+        Navigation.navigate(ROUTES.SETTINGS_TRANSFER_BALANCE);
+    }
+
+    render() {
+        return (
+            <ScreenWrapper>
+                <KeyboardAvoidingView>
+                    <HeaderWithCloseButton
+                        title={this.props.translate('chooseTransferAccountPage.chooseAccount')}
+                        shouldShowBackButton
+                        onBackButtonPress={() => Navigation.goBack()}
+                        onCloseButtonPress={() => Navigation.goBack()}
```

Comment:
This is an intermediate Modal just used for selecting the Payment_metnod for transfer. I believe the user would want to go back to the previous screen instead of cancelling the whole transfer. But let me know if I am wrong.

Reply:
Maybe it is a design question. But why give the user two ways to go back? Usually the pattern is `x` to dismiss and `<` to go back.

Reply:
> Usually the pattern is x to dismiss and < to go back.

Hmm. I agree with this. If the user wants to go back he should use < back key which follows the app's pattern. 

I shall fix it.